src/Twig/Extension.php

```diff
@@ -827,4 +828,16 @@ public function isNumeric($value)
     {
         return is_numeric($value);
     }
+
+    /**
+     * Checks if a given value is an instance of a certain object
+     *
+     * @param object $value
+     * @param string $instance
+     * @return bool
+     */
+    public function isInstanceof($value, $instance)
+    {
+        return $value instanceof $instance;
```

Comment:
Je zou hypothetisch nog wat inputvalidatie kunnen doen op `$value` omdat in PHP < 7.3 een boolean als waarde een fatal error oplevert https://www.php.net/manual/en/language.operators.type.php

Reply:
Goed gespot, ik heb deze regel gewijzigd in `return is_object($value) && $value instanceof $instance`.

Reply:
is het niet handiger om de parameter te typehinten?

Dan kan je een error terug sturen als de parameter value is geen objec

Reply:
Nee, met een typehint krijg je errors die je niet kunt opvangen. Dan gaat het al mis voordat de waarden de methode in gaan. Je wilt juist voorkomen dat je errors krijgt vanuit PHP.

Reply:
ik bedoel juist meer op de manier van dat wanneer je een string doorstuurt dat je dan kan zeggen iets in de tren van deze method heeft een object nodig. 

Dat stuur je dan terug naar de front-end. Zie dat vaak gebeuren en is best wel netjes. Het sluit niet aan op jouw comment hierboven ik hijack alleen de post :P